tsconfig.build.json

```diff
@@ -1,9 +1,8 @@
 {
   "extends": "./tsconfig.json",
+  "include": ["./lib"],
   "compilerOptions": {
     "target": "ES2017",
-    "rootDir": "./lib",
```

Comment:
Since we're already specifying the `include` option, we no longer need to set the `rootDir`.